compiler/src/main/scala/dx/compiler/ApplicationCompiler.scala

```diff
@@ -448,6 +440,8 @@ case class ApplicationCompiler(typeAliases: Map[String, Type],
       case ExecutableKindWorkflowOutputReorg =>
         // The reorg applet requires higher permissions to organize the output directory.
         Some(DxAccess.empty.copy(project = Some(DxAccessLevel.Contribute)))
+      case ExecutableKindWfFragment(_, _, _, _) =>
+        Some(DxAccess.empty.copy(network = Some(Vector("*"))))
```

Comment:
Frag applets have full access, while the other kinds `ExecutableKindWfInputs / ExecutableKindWfOutputs / ExecutableKindWfCustomReorgOutputs` won't.

These kinds of applets are inputs or outputs (default or custom reorg) helpers, so my understanding is that they do not need extra access permission.